pom.xml

```diff
@@ -4364,6 +4364,10 @@
                 <!-- kills mutation in AbstractAutomaticBean -->
                 <param>com.puppycrawl.tools.checkstyle.checks.coding.IllegalTypeCheckTest</param>
               </targetTests>
+              <avoidCallsTo>
+                <avoidCallsTo>org.apache.commons.logging</avoidCallsTo>
```

Comment:
Why avoid calls to Apache logging?

Reply:
By default when `avoidCallsTo` is not used then pit will by default avoid it but when we explicitly applied it will not be considered
https://pitest.org/quickstart/maven/#:~:text=If%20a%20list%20is%20not%20explicitly%20supplied%20then%20PIT%20will%20default%20to%20a%20list%20of%20common%20logging%20packages%20as%20follows

Reply:
this actually required:
```
$ git diff
diff --git a/config/pitest-suppressions/pitest-common-suppressions.xml b/config/pitest-suppressions/pitest-common-suppressions.xml
index 4041118..34258c9 100644
--- a/config/pitest-suppressions/pitest-common-suppressions.xml
+++ b/config/pitest-suppressions/pitest-common-suppressions.xml
@@ -85,6 +85,24 @@
     <sourceFile>Checker.java</sourceFile>
     <mutatedClass>com.puppycrawl.tools.checkstyle.Checker</mutatedClass>
     <mutatedMethod>processFile</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.VoidMethodCallMutator</mutator>
+    <description>removed call to org/apache/commons/logging/Log::debug</description>
+    <lineContent>log.debug(&quot;Exception occurred.&quot;, ex);</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>Checker.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.Checker</mutatedClass>
+    <mutatedMethod>processFile</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.VoidMethodCallMutator</mutator>
+    <description>removed call to org/apache/commons/logging/Log::debug</description>
+    <lineContent>log.debug(&quot;IOException occurred.&quot;, ioe);</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>Checker.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.Checker</mutatedClass>
+    <mutatedMethod>processFile</mutatedMethod>
     <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
     <description>removed call to java/io/IOException::getMessage</description>
     <lineContent>new String[] {ioe.getMessage()}, null, getClass(), null));</lineContent>
diff --git a/pom.xml b/pom.xml
index 70f3a90..7be5ccf 100644
--- a/pom.xml
+++ b/pom.xml
@@ -4365,7 +4365,6 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.coding.IllegalTypeCheckTest</param>
               </targetTests>
               <avoidCallsTo>
-                <avoidCallsTo>org.apache.commons.logging</avoidCallsTo>
                 <avoidCallsTo>com.puppycrawl.tools.checkstyle.utils.UnmodifiableCollectionUtil</avoidCallsTo>
               </avoidCallsTo>
               <excludedMethods>

```

Reply:
Resolved